sdcm/tester.py

```diff
@@ -800,7 +803,9 @@ def is_encrypt_keys_needed(self):
         )
 
     def prepare_kms_host(self) -> None:
-        if self.params.is_enterprise and self.params.get('cluster_backend') == 'aws' and not self.params.get('scylla_encryption_options'):
+        if (self.params.is_enterprise and ComparableScyllaVersion(self.params.scylla_version) >= '2023.2.0~rc0' and
+                self.params.get('cluster_backend') == 'aws'
```

Comment:
Please, use the `and` operator as a first word on the continuation line.
It is common recommendation

Reply:
does black enforces that ?

Reply:
anyhow I've fixed it.